### PR TITLE
Shorter ash storms

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I'll think about redoing how they work later but this is easier for now

### CODE REVIEW

File: code/modules/mining/lavaland/weather_types.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 	start_up_time = 300 //30 seconds
 	start_up_message = "The wind begins to pick up. Seek shelter."
 	duration_lower = 60 //1 minute
-	duration_upper = 180 //3 minutes
+	duration_upper = 120 //3 minutes
```
Comment (Cheridan) [inline code/modules/mining/lavaland/weather_types.dm]: comment is now inaccurate!!!!

Comment (KorPhaeron) [inline code/modules/mining/lavaland/weather_types.dm]: It's a beta

Comment (RemieRichards) [inline code/modules/mining/lavaland/weather_types.dm]: update comments or don't put them in.

Comment (lzimann) [inline code/modules/mining/lavaland/weather_types.dm]: forgot to change commentary but not a big deal

Comment (Zilenan91) [inline code/modules/mining/lavaland/weather_types.dm]: literally unplayable

Comment (duncathan) [inline code/modules/mining/lavaland/weather_types.dm]: Will NOT be merged unless you fix this. Come on dude, get your shit together. You should know better.

Comment (Cheridan) [inline code/modules/mining/lavaland/weather_types.dm]: i can't believe i promoted this man to a position of authority

Comment (Zilenan91) [inline code/modules/mining/lavaland/weather_types.dm]: he smart, he loyal

Comment (Iamgoofball) [inline code/modules/mining/lavaland/weather_types.dm]: :ok_hand:

Comment (Archie700) [inline code/modules/mining/lavaland/weather_types.dm]: >>>our design lead

**Comment by TechnoAlchemisto** [comment]
:(

**Pull request merged**